### PR TITLE
py-jsonpath-ng: version bump

### PR DESCRIPTION
**Pull request by heerener**
Pushing internal version bump to 1.5.3, adding 1.6.9 since I'm here anyway.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-jsonpath-ng/package.py [hunk 1]
```diff
@@ -15,9 +15,11 @@ class PyJsonpathNg(PythonPackage):
     homepage = "https://github.com/h2non/jsonpath-ng"
     pypi = "jsonpath-ng/jsonpath-ng-1.5.2.tar.gz"
 
+    version("1.6.9", sha256="5483f8e9d74c39c9abfab554c070ae783c1c8cbadf5df60d561bc705ac68a07e")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/py-jsonpath-ng/package.py]: There is no `1.6.9` version, I think you mean `1.6.0` (which DOES have that sha256).

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Confirmed one of the two version sha256.  There is no `1.6.9` so please correct the version directive for `1.6.0`.

File: var/spack/repos/builtin/packages/py-jsonpath-ng/package.py (hunk 1 above)
Comment (heerener) [reply var/spack/repos/builtin/packages/py-jsonpath-ng/package.py]: Oh, thanks for catching that! Indeed, typo.

**Review by tldahlgren — APPROVED** [review]
Thanks for the correction.  Will still defer merge decision to @adamjstewart as the python package guru.

**Review by manuelakuhn — APPROVED** [review]

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**